pages/help/_slug.vue

```diff
@@ -4,33 +4,45 @@
             :title="page.title"
             :text="page.summary"
         />
-        <RichText
-            v-if="page.richText"
-            class="content"
-            :rich-text-content="page.richText"
-        />
-        <DividerWayFinder color="help" />
-        <div
+        
+        <section-wrapper>
+            <RichText
+                v-if="page.richText"
+                :rich-text-content="page.richText"
+            />
+        </section-wrapper>
+        
+        <section-wrapper theme="divider">
+            <DividerWayFinder color="help" />
+        </section-wrapper>
+        
+        <section-wrapper
             v-for="(block, index) in page.helpTopicBlocks"
             :key="`HelpTopicBlocksKey${index}`"
             class="help-topic-section"
         >
-            <simple-cards
-                class="help-topic-block"
-                :section-title="block.sectionTitle"
-                :section-summary="block.sectionSummary"
-                :items="block.associatedEntries"
-            />
-            <DividerWayFinder
-                color="help"
-                class="help-topic-divider"
-            />
-        </div>
+            <section-wrapper>
+                <simple-cards
+                    class="help-topic-block"
```

Comment:
Looks like we can remove [this class](https://github.com/UCLALibrary/library-website-nuxt/blob/3421a8bba479fb144a84c421b0cf42538a8a6917/pages/help/_slug.vue#L79-L82) since it just sets margins/padding and max-width